frontend/src/__tests__/cypress/cypress/pages/modelServing.ts

```diff
@@ -243,6 +243,14 @@ class InferenceServiceModal extends ServingModal {
     return this.find().findByTestId('service-account-form-name');
   }
 
+  findSpecificServiceAccountNameInput(index: number) {
+    return this.find().findAllByTestId('service-account-form-name').eq(index);
+  }
```

Comment:
Maybe this will make more sense?
```suggestion
  findServiceAccountIndex(index: number) {
    return this.find().findAllByTestId('service-account-form-name').eq(index);
  }
```